entry_types/scrolled/package/src/contentElements/externalLinkList/editor/Sidebar.module.css

```diff
@@ -0,0 +1,51 @@
+.container {
+  margin-top: 30px;
+}
+.add{
+  border: 1px solid #1c86fe;
+  border-radius: 3px;
+  box-shadow: inset 0 1px 0 0 #b6d5f8;
+  color: white !important;
+  display: inline-block;
+  font-size: 11px;
+  font-weight: bold;
+  background-color: #6aacf7;
+  background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #6aacf7), color-stop(100%, #2b8efe));
+  background-image: -webkit-linear-gradient(#6aacf7, #2b8efe);
+  background-image: linear-gradient(#6aacf7, #2b8efe) !important;
+  padding: 7px 18px;
+  text-decoration: none !important;
+  text-shadow: 0 1px 0 #067bff;
+  background-clip: padding-box;
+  padding-top: 5px;
+  padding-bottom: 4px;
+  padding-left: 12px
+}
+
+.add::before{
+  font-family: 'entypo';
+  content: "➕"
+}
+
+.add:hover:not(:disabled):not(.disabled) {
+  box-shadow: inset 0 1px 0 0 #87baf4;
+  cursor: pointer;
+  background-color: #559ff2;
+  background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #559ff2), color-stop(100%, #1d86fc));
```

Comment:
I'm still wondering about the browser-specific CSS used here. Have you tried this in Firefox (`-moz-gradient` and `-moz-linear-gradient`)?